hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -2420,8 +2431,6 @@ private void startActiveMasterManager(int infoPort) throws KeeperException {
       } else {
         abort("Unhandled exception. Starting shutdown.", t);
       }
-    } finally {
-      status.cleanup();
```

Comment:
Sorry, maybe I didn't understand the config. Can you explain what the implication of this change is? Maybe show screenshot?

It would be nice to keep the main master UI clean if possible, but keep the full history in your new startup progress UI for lifetime.

Reply:
If "hbase.master.ignore.clear.startup.status" is off(default on), when master startup successfully, all the steps info base by the phased monitored tasks will be cleared. This means we can see the starup progress during the master becoming active, but after active the phased tasks is cleared, we can see nothing about the startup progress then through the UI.
We remove this config then the startup phased monitored tasks are kept in the memory, then whenever open the starup UI, the detailed phased info is showed. This makes the destination of the issue more clear, and is definitely  different to the previous startup monitor task.

Reply:
Thanks that clarifies. That's what we want -- retain history for process lifetime so it's always visible on startupProgress UI. So removing config is good IMO, thanks. 

I imagine the parent task will be removed from main master UI since we markComplete at end.